x-pack/solutions/observability/plugins/streams/server/lib/streams/client.ts

```diff
@@ -430,6 +430,21 @@ export class StreamsClient {
     }
   }
 
+  private validateIsChildOf(parent: string, destination: string) {
```

Comment:
Nit/question: Do we make any distinction between validating and **asserting** in the plugin? 

From the outside, this looks like an assertion (stop running further code if this condition is not met) compared to a validation (which has some outcome that is then acted on).

Reply:
Maybe assertion would be the better naming, no strong opinion.